dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/handlers/AndroidLogsHandler.kt

```diff
@@ -0,0 +1,40 @@
+package com.datadog.android.tracing.internal.handlers
+
+import com.datadog.android.log.Logger
+import datadog.opentracing.LogsHandler
+
+internal class AndroidLogsHandler(
+    val logger: Logger
+) : LogsHandler {
+
+    // region Span
+    override fun log(timestampMicroseconds: Long, fields: Map<String, *>) {
+        // not sure that we should do something with the timestamp here ???
+        logFields(fields)
+    }
+
+    override fun log(event: String) {
+        logger.v(event)
+    }
+
+    override fun log(timestampMicroseconds: Long, event: String) {
+        // not sure that we should do something with the timestamp here
+        logger.v(event)
+    }
+
+    override fun log(map: Map<String, *>) {
+        logFields(map)
+    }
+
+    // endregion
+
+    // region Internal
+
+    private fun logFields(fields: Map<String, *>) {
```

Comment:
this doesn't log anything at all, besides the fields should be logged as attributes IMHO (tags might impact the value as some characters are illegal in tags). 
I'd implement it as follow:
```kotlin
private fun logFields(fields: Map<String, *>) {
    logger.v("span event", attributes = fields)
}
```